app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/askforprice/AskForPriceInfoContent.kt

```diff
@@ -23,37 +24,40 @@ fun IntroContent(
 ) {
     val baseMargin = dimensionResource(R.dimen.base_margin)
     val baseMarginDouble = dimensionResource(R.dimen.base_margin_double)
-    val baseMarginQuadruple = dimensionResource(R.dimen.base_margin_quadruple)
 
     Column(
         modifier = Modifier
             .padding(baseMarginDouble)
-            .fillMaxWidth(),
-        verticalArrangement = Arrangement.spacedBy(baseMarginDouble)
+            .fillMaxWidth()
+            .fillMaxHeight(),
```

Comment:
```suggestion
            .fillMaxSize(),
```
That should probably be functionally equivalent afaik.